Kernel/Graphics/Console/BootFramebufferConsole.cpp

```diff
@@ -30,7 +43,9 @@ void BootFramebufferConsole::clear(size_t x, size_t y, size_t length)
 
 void BootFramebufferConsole::clear_glyph(size_t x, size_t y)
 {
+#if !ARCH(AARCH64)
```

Comment:
Please put a `FIXME` note here to point that is needed to be fixed.

Reply:
Instead of putting a FIXME at all the callsites of the SpinLock related stuff, we can just always return true in `is_locked()`, and put the FIXME there instead.

Reply:
@FireFox317 Done!